PurchasesTests/SubscriberAttributes/SubscriberAttributeTests.swift

```diff
@@ -13,44 +13,24 @@ class SubscriberAttributeTests: XCTestCase {
     func testInitWithKeyValueSetsRightValues() {
         let now = Date()
         let dateProvider = MockDateProvider(stubbedNow: now)
-        let subscriberAttribute = RCSubscriberAttribute(key: "a key",
-                                                        value: "a value",
-                                                        dateProvider: dateProvider)
+        let subscriberAttribute = SubscriberAttribute(withKey: "a key",
+                                                      value: "a value",
+                                                      dateProvider: dateProvider)
         expect(subscriberAttribute.key) == "a key"
         expect(subscriberAttribute.value) == "a value"
         expect(subscriberAttribute.setTime) == now
         expect(subscriberAttribute.isSynced) == false
     }
 
-    func testInitWithDictionarySetsRightValues() {
```

Comment:
Moved to cache tests since the method that is being tested now exists in the caching class.